CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ Same changes as in `@mui/x-date-pickers@8.0.0-beta.2`.
 
 #### `@mui/x-charts@8.0.0-beta.2`
 
+- [charts] Memoize axes and series with default (#17156)
```

Comment:
We are missing the name of the dev (and alphabetical order would be awesome but I'm nitpicking here)